engines/dfc_provider/spec/controllers/dfc_provider/api/enterprises_spec.rb

```diff
@@ -41,6 +41,14 @@
               end
             end
           end
+
+          context 'given with a wrong id' do
+            before { api_get :show, id: 999 }
+
+            it 'returns 404' do
+              expect(response.status).to eq 404
```

Comment:
I think you can/should use :not_found instead of 404

Reply:
I meant the symbol :not_found :-) like this:
              expect(response.status).to eq :not_found